QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
+QUEST_20150511_001989	$Helping reconstruct Fedimian may be more meaningful than pottery. {nl}But people need something to soothe their hearts.
+QUEST_20150511_001990	$If I had the diary, I could continue my adventure, but I wasn't selected… {nl}But if it were you…
+QUEST_20150511_001991	$If you cleverly use the dry air that forms around severe colds, you'd be able to freeze any object even in a desert. {nl}However, if you can't receive the blessing from the Goddess, it won't be possible to make that kind of air.
+QUEST_20150511_001992	$I'll report the damages. {nl}Chasing a Sparnas, a gigantic bird-type creature, will be hard. Despite being large, they are rather agile. {nl}It usually goes after the supplies, but I can't imagine nor find out why. I'm guessing someone's playing a trick.
+QUEST_20150511_001993	$Grain pocket was discovered in the ashes of the supplies. {nl}They smell so bad and old they can't be used for food at all. {nl}Please check the supplies for anything missing.
+QUEST_20150511_001994	$Defeat all the monsters nearby. {nl}The vicious energy in the vine forest is constantly creating new monsters. {nl}But you should first defeat most of them before you face Ironbaum. {nl}Even a small factor could ruin the big task. {nl}Also, you could try threatening the monsters so they can't do what they want to do.
+QUEST_20150511_001995	$(Everything is scribbled so poorly that they can't be distinguished from letters or drawings.)
+QUEST_20150511_001996	$The facility that belonged to the workers who worked on the Mausoleum of Zachariel suddenly shut down. {nl}It's not like they left because it was useless but they all just vanished all at once. {nl}Maybe it is related to the irregularly big size of the tomb and its complicated composition {nl}even after considering the fact that it was the grave to commemorate the greatest king in history.
+QUEST_20150511_001997	$King Zachariel had left many arrangements for future generations. {nl}We and the Jonas family were one of them. {nl}But the meaning of it is only known by the Goddess. {nl}At least we know that the will of the king that we are protecting and the secret of the tomb indicates one person.
+QUEST_20150511_001998	$Trees become bigger as years pass. {nl}We can only see Woodman here but if you think of the large trees or the trees that stand in the old capital city… {nl}You can't help but think of the existence of even larger trees. {nl}After the disaster occurred, many plants turned into monsters so we can't ever rule out that possibility.
+QUEST_20150511_001999	$(There are barely legible words written here.) {nl}Euros could not hold his breath any more so he ended up inhaling the spores and died. {nl}Mushwort hunt people with the cruelest method I've ever seen. {nl}They walks around our gathering places and shakes out the mushrooms that would release spores.
```

Comment:
If Mushwort is a monster name, then it should be "Mushworts" here. Also: "They walks around our gathering places and shakes out", should be "They walk" and "shake out".